bin/dar

```diff
@@ -5,6 +5,30 @@ package Cyrus::Docker::dar;
 
 use utf8;
 
+BEGIN {
```

Comment:
a cpanfile would be nice too but this works

Reply:
I feel like there can/will be a better install/setup story later :)